edx/edx-analytics-data-api
# ENT-1718:  Django Admin updates for user role based permissions

__Description:__

This PR removes the `ENABLE_ADMIN_SITE` so that admin is enabled by default. It also adds the new `enterprise_data_roles` app into `INSTALLED_APPS`.

**Testing Instructions:**

1. Create a superuser for django admin
2. Run `python manage.py runserver`
2. Goto http://127.0.0.1:8000/admin
3. Open `Enterprise data role assignments` model
4. Create a role assignment and it should save successfully
5. Open `Switches` model under `WAFFLE` app
6. Create/Update/Delete switch should work as expected
